src/System Application/App/Extension Management/src/ExtensionMarketplace.Page.al

```diff
@@ -19,6 +19,9 @@ page 2502 "Extension Marketplace"
     ApplicationArea = All;
     UsageCategory = Administration;
     Editable = false;
+    ObsoleteState = Pending;
```

Comment:
Shouldn't you introduce CLEAN tags and handle the references for this page? Or will you handle it in the next PR?